### PR TITLE
Add kssd

### PR DESCRIPTION
**Pull request by druvus**
Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it. Note that if you are not already a member of the bioconda project (meaning that you can't add this label), please ping `@bioconda/core` so that your PR can be reviewed and merged (please note if you'd like to be added to the bioconda project). Please see https://github.com/bioconda/bioconda-recipes/issues/15332 for more details.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by druvus** [comment]
Only Linux is supported according to their [documentation](https://github.com/yhg926/public_kssd) so skipping osx.

> Kssd run on linux system, currently not support for MacOS and Windows OS.

File: recipes/kssd/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+build:
+  number: 0
+  skip: True  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - zlib
```
Comment (dpryan79) [inline recipes/kssd/meta.yaml]: Is this statically linking zlib in?

Comment (dpryan79) [reply recipes/kssd/meta.yaml]: It looks like this makes system calls to gzip (it incorrectly reads in the zlib headers and then doesn't use them). This is fine then.

**Comment by druvus** [comment]
thanks @dpryan79

**Pull request merged**